docs/graph/controls/PeoplePicker.md

```diff
@@ -9,7 +9,7 @@ dev_langs:
 
 # PeoplePicker XAML Control
 
-The [PeoplePicker](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.graph.controls.peoplepicker) searches for people and renders the list of results from Microsoft Graph. By default, the component will search across all people.
+The [PeoplePicker](https://docs.microsoft.com/dotnet/api/microsoft.toolkit.uwp.ui.controls.graph.peoplepicker) searches for people and renders the list of results from Microsoft Graph. By default, the component will search across all people.
```

Comment:
That's the old link, the new link was never published due to confusion in how our docs work. I'm trying to fix that in #344 , but it's still not working.

Reply:
@mrlacey you'll see the original link is working now. 🎉 That was fixed as part of #347